data/sql/updates/pending_db_world/reckoning-downrank.sql

```diff
@@ -0,0 +1 @@
+UPDATE `spell_proc_event` SET `entry` = -20177 WHERE `entry` = 20182 AND `procEx` = 262211;
```

Comment:
```suggestion
UPDATE `spell_proc_event` SET `entry` = -20177 WHERE `entry` = 20182;
```
Entry is the primary key with none else, so there can only be one record. No need to check for procEx